passenger_exporter_test.go

```diff
@@ -174,6 +179,7 @@ func TestUpdateProcessIdentifiers(t *testing.T) {
 				Process{PID: "jkl"},
 				Process{PID: "lmn"},
 			},
+			9,
```

Comment:
How have you come up with `9` here?

Reply:
See https://github.com/Intellection/passenger-exporter/pull/14#discussion_r394157681.